mqtt/mqtt-bridge/src/persist/waking_state/mod.rs

```diff
@@ -6,6 +6,7 @@ use mqtt3::proto::Publication;
 use crate::persist::{Key, PersistError};
 
 pub mod memory;
+#[allow(dead_code)]
```

Comment:
Thought one of these was already here, let me figure out if I put another one by accident...